scripts/run.sh

```diff
@@ -192,37 +195,38 @@ EOF
 # https://github.com/ava-labs/avalanche-network-runner
 # TODO: use "go install -v github.com/ava-labs/avalanche-network-runner/cmd/avalanche-network-runner@v${NETWORK_RUNNER_VERSION}"
 NETWORK_RUNNER_VERSION=1.0.12
```

Comment:
Can we take in the network runner version from `scripts/versions.sh` and keep it up to date there?

Reply:
aaahhh I requested that it in #155 but seems forgot to actually send the review...  geez...
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/18367708/180770909-8fc19f26-7a41-49cb-b606-c3ffdcb18601.png">